components/Opportunities/index.jsx

```diff
@@ -0,0 +1,105 @@
+import Image from 'next/image';
+import PropTypes from 'prop-types';
+import {
+  Button, Card, Col, Row, Typography,
+} from 'antd';
+import styled from 'styled-components';
+import opportunities from './opportunities.json';
+
+const LAUNCH_CONTACT_URL = 'https://t.me/pahlmeyer';
+
+const StyledCard = styled(Card)`
+  width: 100%;
+  height: 100%;
+  align-items: center;
+  .ant-card-body {
+    min-height: 100%;
+    display: flex;
+  }
+  margin-bottom: 24px;
+`;
+
+const OpportunityCard = ({
+  agentName,
+  agentDescription,
+  project,
+  image,
+  background,
+}) => (
+  <StyledCard>
+    <Row style={{ columnGap: '12px', width: '100%', minHeight: '100%' }}>
```

Comment:
better to use gutter instead of column gap, it's what we're used to https://ant.design/components/grid#grid-demo-gutter; can you also check if both `width: '100%', minHeight: '100%'` are needed, looks like all works without them?

Reply:
I've added gutter and removed minheight but width:100% is still needed